iothub/device/src/Microsoft.Azure.Devices.Client.csproj

```diff
@@ -121,12 +124,4 @@
     <PackageReference Include="Microsoft.AspNet.WebApi.Client" Version="5.2.3" />
     <PackageReference Include="Microsoft.Owin" Version="4.0.0" />
   </ItemGroup>
-
-  <ItemGroup>
-    <!-- FXCop -->
-    <PackageReference Condition=" '$(Configuration)' == 'Debug' " Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="3.3.1">
```

Comment:
We used to have this on only in Debug. I don't quite think that is what we'd have done if we started fresh. Any opinions?

Reply:
I personally think we should add it for both debug and release unless there is a specific drawback to having it enabled.